OpenRA.Mods.Common/Traits/BotModules/Squads/States/AirStates.cs

```diff
@@ -65,14 +65,34 @@ protected static Actor FindDefenselessTarget(Squad owner)
 			var map = owner.World.Map;
 			var dangerRadius = owner.SquadManager.Info.DangerScanRadius;
 			detectedEnemyTarget = null;
-			var x = (map.MapSize.X % dangerRadius) == 0 ? map.MapSize.X : map.MapSize.X + dangerRadius;
-			var y = (map.MapSize.Y % dangerRadius) == 0 ? map.MapSize.Y : map.MapSize.Y + dangerRadius;
-
-			for (var i = 0; i < x; i += dangerRadius * 2)
+			var initialMaxX = (map.MapSize.X % dangerRadius) == 0 ? map.MapSize.X : map.MapSize.X + dangerRadius;
+			var initialMaxY = (map.MapSize.Y % dangerRadius) == 0 ? map.MapSize.Y : map.MapSize.Y + dangerRadius;
```

Comment:
These calculations assume that `j` is a multiple of `dangerRadius`, which is no longer true thanks to the random starting offset!